Enable configurable CC and XROOT.
Add missing $(LIBS) to link and remove strip.
Add yeahwm.1 to install target.

--- a/Makefile	2004-11-21 19:11:04.000000000 +0100
+++ b/Makefile	2015-10-25 23:20:45.306617941 +0100
@@ -1,11 +1,11 @@
 # If you don't use CC 
-CC       = gcc
+CC       ?= gcc
 
 # Edit this line if you don't want yeahwm to install under /usr.
 # Note that $(DESTDIR) is used by the Debian build process.
 prefix = $(DESTDIR)/usr
 
-XROOT    = /usr/X11R6
+XROOT    ?= /usr/X11R6
 INCLUDES = -I$(XROOT)/include
 LDPATH   = -L$(XROOT)/lib
 LIBS     = -lX11
@@ -55,7 +55,6 @@
 all: yeahwm
 
 yeahwm: $(OBJS)
-	$(CC) $(CFLAGS) $(OBJS) -o $@ $(LDFLAGS)
+	$(CC) $(CFLAGS) $(OBJS) -o $@ $(LDFLAGS) $(LIBS)
- 	strip yeahwm
 
 allinone:
@@ -71,7 +71,7 @@
 	if [ -f yeahwm.exe ]; then mv yeahwm.exe yeahwm; fi
 	mkdir -p $(prefix)/bin $(prefix)/share/man/man1
 	install -s yeahwm $(prefix)/bin
-	#install yeahwm.1 $(prefix)/share/man/man1
+	install yeahwm.1 $(prefix)/share/man/man1
 	#gzip -9 $(prefix)/share/man/man1/yeahwm.1
 
 install: doinstall
